tests/test_lnchannel.py

```diff
@@ -138,6 +141,8 @@ def create_test_channels(*, feerate=6000, local_msat=None, remote_msat=None,
     funding_sat = ((local_msat + remote_msat) // 1000) if local_msat is not None and remote_msat is not None else (bitcoin.COIN * 10)
     local_amount = local_msat if local_msat is not None else (funding_sat * 1000 // 2)
     remote_amount = remote_msat if remote_msat is not None else (funding_sat * 1000 // 2)
+    local_max_inflight = local_amount if local_max_inflight is None else local_max_inflight
+    remote_max_inflight = remote_amount if remote_max_inflight is None else remote_max_inflight
```

Comment:
why use local_amount and remote_amount as default? one of these values can get arbitrarily low

Reply:
indeed, should've been `funding_sat`